src/primitives/threshold.rs

```diff
@@ -0,0 +1,310 @@
+// SPDX-License-Identifier: CC0-1.0
+
+//! Thresholds
+//!
+//! Miniscript
```

Comment:
```suggestion
//! A generic (k,n)-threshold type.
```

Literally the only thing that was better from #605 :)

Reply:
Lol, oh, right :) I should finish this doccomment.